conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureDoubleList.java

```diff
@@ -69,4 +75,13 @@ public void clear() {
     public Double set(int index, Double element) {
         return delegate.set(index, element);
     }
+
+    static final class Serializer extends JsonSerializer<ConjureDoubleList> {
+        @Override
+        public void serialize(ConjureDoubleList val, JsonGenerator gen, SerializerProvider _serializer)
+                throws IOException {
+            double[] doubles = val.delegate.toArray();
+            gen.writeArray(doubles, 0, doubles.length);
```

Comment:
Does this impact our use of jackson-databind configuration options like excluding empty collections? https://github.com/palantir/conjure-java/blob/7ed0bdac3e25f5f567325cbf2ae580168f2cc99b/conjure-java/src/main/java/com/palantir/conjure/java/cli/ConjureJavaCli.java#L212-L218

Reply:
I suspect we may be better off adding a `@JsonValue` annotation on the `toArray` method. A custom unmodifiable wrapper will need to be implemented to provide a `@JsonValue` annotated `toArray` method in a place that is visible to jackson (the outermost type)